### ISSUE TITLE
Use BLAST DB api

### ISSUE THREAD

**Issue by czabriskie** [OtimusOne/AFPAP#2]
Instead of pulling entire blast database. Maybe we can use an API instead.

**Comment by OtimusOne** [comment]
Unfortunately is seem the [NCBI API](https://ncbi.github.io/blast-cloud/) is no longer supported, they now offer documentation for running BLAST from your own cloud server.

I am not aware of other APIs that allow using BLAST on such large databases as UniRef90. [MMseqs2](https://search.mmseqs.com/) might be an alternative, but they also seems to have trouble with resources at the moment.

I had briefly tested MMseqs2 locally once before and I couldn't get nearly the same amount of hits as BLAST did, though I might not have used the proper arguments. There could also be other ways of computing the sequence conservation.

**Comment by czabriskie** [comment]
Yeah I feel this is something that would make the package a lot more user friendly. I think downloading these bits of data is what takes the longest and requires the most commitment
